apparmor.d/groups/akonadi/akonadi_mailfilter_agent

```diff
@@ -16,6 +16,7 @@ profile akonadi_mailfilter_agent @{exec_path} {
   include <abstractions/mesa>
   include <abstractions/nameservice-strict>
   include <abstractions/qt5>
+  include <abstractions/user-tmp>
```

Comment:
Could you explain what tmp file is needed. It is usually not needed to give full access to user-tmp.

Reply:
I'll check that.